windows/runner/win32_window.cpp

```diff
@@ -0,0 +1,245 @@
+#include "win32_window.h"
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase